Libs/DICOM/Widgets/ctkDICOMServerNodeWidget2.h

```diff
@@ -92,7 +92,7 @@ class CTK_DICOM_WIDGETS_EXPORT ctkDICOMServerNodeWidget2 : public QWidget
   Q_INVOKABLE int serversCount();
   Q_INVOKABLE ctkDICOMServer* getNthServer(int id);
```

Comment:
`get` prefix is not Qt style

```suggestion
  Q_INVOKABLE ctkDICOMServer* server(int id);
```